jupyterhub_ssh/tests/test_terminado.py

```diff
@@ -0,0 +1,45 @@
+from aiohttp import ClientSession
+import os
+import pytest
+from yarl import URL
+import sys
+
+from notebook.tests.launchnotebook import NotebookTestBase
+
+from ..terminado import Terminado
+
+# Mark all tests in this file as asyncio
+pytestmark = pytest.mark.asyncio
+
+@pytest.fixture
+async def notebook():
+    # Start the notebook
+    notebook = NotebookTestBase()
+    notebook.setup_class()
+    notebook.wait_until_alive()
+    yield notebook
+    notebook.teardown_class()
```

Comment:
I see that jupyter_server seem to have an equivalent. I'm not sure if its relevant to transition to that or not though.

Related issue: https://github.com/elyra-ai/elyra/issues/831#issue-674537628

Related fixture from jupyter_server: https://github.com/jupyter-server/jupyter_server/blob/7d2f04f30e06e6efed90072df19824d416cb7129/jupyter_server/pytest_plugin.py#L185-L210